terraform/aws/modules/bastion/main.tf

```diff
@@ -90,6 +90,7 @@ resource "aws_instance" "bastion" {
   instance_type        = var.bastion_ec2_type
   ami                  = data.aws_ami.bastion_ami.id
   key_name             = var.key_pair
+  disable_api_termination = true
```

Comment:
no hard coded values in main, please declair a new variable in variables.tf, default should be true